packages/vite/src/client/error.ts

```diff
@@ -0,0 +1,266 @@
+import type { RawSourceMap } from 'source-map'
+import { SourceMapConsumer } from 'source-map'
```

Comment:
Using source-map in browser seems a bit risky, 0.6 works fine but 0.7 introduced a wasm dependency, and 0.8 (beta) [introduces a 300k whatwg-url one](https://github.com/mozilla/source-map/issues/400).

Reply:
this method could probably be used when we have moved over to `source-map-js`